docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_RNNT.md

```diff
@@ -0,0 +1,93 @@
+# Convert ONNX* RNNT to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_RNNT}
+
+This instruction convers conversion of RNN-t model from [MLCommons](https://github.com/mlcommons) repository. 
+If you already have a downloaded MLCommons repository you can go right away to **Step 5**.
+
+**Step 1**. Clone the MLCommons repository for inference:
+```bash
+git clone https://github.com/mlcommons/inference
+```
+
+**Step 2**. create a folder for storing pre-trained model and onnx files:
+```bash
+mkdir rnnt_for_openvino 
+cd rnnt_for_openvino
+```
+
+**Step 3**. Download pre-trained weights for PyTorch implementation from https://zenodo.org/record/3662521#.YG21DugzZaQ:
+For UNIX* like systems you can use wget
+```bash
+wget https://zenodo.org/record/3662521/files/DistributedDataParallel_1576581068.9962234-epoch-100.pt
+```
+
+**Step 4**. Install required python packages:
+```bash
+pip install torch toml
+```
+
+**Step 5**. Save the following script in the folder created at the **Step 2**:
+Before running the script specify path where you cloned MLCommons repository so that script could find `config_toml`. 
```

Comment:
minor note: possibly use path to MLCommons as enviroment variable that user should specify to avoid script changing by user and use in PYTHONPATH

Reply:
You are right. The old method was too involved. Instead of changing environment variable added subdirectory with PyTorch clases into `sys.path` in the script. So user can just type python `export_to_onnx.py` in the current directory, no need to specify path to MLComons if user followed step 1.